libs/labelbox/tests/data/annotation_import/test_mea_prediction_import.py

```diff
@@ -233,19 +230,13 @@ def test_create_from_label_objects(
                     ),
                     ObjectAnnotation(
                         name="bbox",
-                        extra={
```

Comment:
This could be breaking but want a second opinion. The extra field breaks the annotation if a customer decides to add it. It is not shown that they need to do this in the docs, and I don't see why they would. The old complicated logic removed these but to simplify this I think it is best to just not support this type of extra key